### ISSUE TITLE
Pypi maintainer rights

### ISSUE THREAD

**Issue by thet** [collective/collective.multilingual#6]
@malthe I'm working on Plone 5.2+ compatibility.
I think this package has still relevance as it does make fewer assumpions on how the content should be structured.

Could you give me maintainer rights on pypi? My pypi user name is: thet

https://pypi.org/project/collective.multilingual/

By the way: did this package start as a fork of plone.multilingual/plone.app.multilingual?

**Comment by malthe** [comment]
Hi Johannes. I have granted you the "Maintainer" role. Sorry for the delay – my Hacintosh ^D^D^D Mac suffered a bit of a stroke over New Year's.

Did I start it as a fork? No – this was developed independently. I did not think PAM was very mature at the time and was overly complicated while at the same time having rather poor usability. A longer explanation is available in the https://github.com/collective/collective.multilingual#frequently-asked-questions.
